src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -79,6 +81,9 @@ export class DictionaryOverview extends Component {
   handleHide = () => this.setState({ showEditModal: false });
   handleShow = () => this.setState({ showEditModal: true });
 
+  handleHideSub = () => this.setState({ showSubModal: false });
```

Comment:
Shouldn't you set the state url back to `''`? Just asking because the handleShowSub edits it.